templates/base.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
```

Comment:
It is picky, but I would prefer one tab size all over the project. You can find it in Settings -> Editor -> Code style -> HTML